docs/installation/server-config/security.rst

```diff
@@ -22,6 +25,83 @@ file and then :ref:`start the KSQL server <start_ksql-server>` with your configu
 .. contents:: Table of Contents
     :local:
 
+Configuring KSQL for BASIC HTTP Authentication
+----------------------------------------------
+KSQL can be configured to require users to authenticate using a username and password via the BASIC
+HTTP authentication mechanism.
+
+NOTE: If using BASIC authentication it is highly recommended that you configure the KSQL server to
+use SSL to secure communication, as the BASIC protocol passes credentials in plain text.
+
+Use the following to configure the KSQL server to require authentication:
+
+.. code:: bash
+
+    authentication.method=BASIC
+    authentication.roles=some-ksql-cluster-id
+    authentication.realm=KsqlServer-Props
+    java.security.auth.login.config=/path/to/the/jaas_config.file
+
+The ``authentication.roles`` config defines a comma separated list of user roles. To be authorized
+to use the KSQL server an authenticated user must belong to at least one of these roles.
+
+The ``authentication.realm`` config must match a section with in ``jaas_config.file``, which will
+define how the server will authenticate users:
```

Comment:
"which defines how the server authenticates users..."

Reply:
done